index.js

```diff
@@ -35,15 +35,181 @@ if (window.innerWidth < 767) {
 
 /* Form Validation Section */
 
-const email = document.getElementById('mail');
-const form = document.getElementById('form1');
-
-form.addEventListener('submit', (event) => {
-  event.preventDefault();
-  if (email.value !== email.value.toLowerCase()) {
-    document.getElementById('error').innerText = 'Error, Email must be lowercase. Form is not sent';
-  } else {
-    document.getElementById('error').innerText = '';
-    form.submit();
-  }
-});
+// const email = document.getElementById('mail');
+// const form = document.getElementById('form1');
+
+// form.addEventListener('submit', (event) => {
+//   event.preventDefault();
+//   if (email.value !== email.value.toLowerCase()) {
+//     document.getElementById('error').innerText = 'Error, Email must be lowercase.';
+//   } else {
+//     document.getElementById('error').innerText = '';
+//     form.submit();
+//   }
+// });
```

Comment:
- [ ] Kindly remove non-explanatory commented code as it helps your codebase be cleaner